src/main/java/com/redhat/devtools/lsp4ij/usages/LSPExternalReferencesFinder.java

```diff
@@ -70,22 +70,29 @@ public static void processExternalReferences(@NotNull PsiFile file,
                                                  int offset,
                                                  @NotNull SearchScope searchScope,
                                                  @NotNull Processor<PsiReference> processor) {
-        VirtualFile virtualFile = file.getVirtualFile();
-        if (virtualFile != null) {
-            Document document = LSPIJUtils.getDocument(virtualFile);
-            TextRange wordTextRange = document != null ? LSPIJUtils.getWordRangeAt(document, file, offset) : null;
-            if (wordTextRange != null) {
-                LSPPsiElement wordElement = new LSPPsiElement(file, wordTextRange);
-                String wordText = wordElement.getText();
-                if (StringUtil.isNotEmpty(wordText)) {
-                    processExternalReferences(
-                            file,
-                            wordText,
-                            wordTextRange,
-                            searchScope,
-                            ProgressManager.getInstance().getProgressIndicator(),
-                            processor
-                    );
+        // Only if enabled by at least one of the file's language servers
```

Comment:
If you look at this with whitespace-only changes hidden, you'll see that this logic is now gated by a feature flag check. This ensures that **all** callers must pass this gate including `LSPUsageSearcher` and `LSPRenameRefactoringDialog` which wasn't disabled for 0.12.0.